src/OAuth2/Encryption/EncryptionInterface.php

```diff
@@ -4,8 +4,8 @@
 
 interface EncryptionInterface
 {
-    public function encode($payload, $key, $algorithm = null);
-    public function decode($payload, $key, $algorithm = null);
+    public function encode($payload, $key, $algorithm = null, $keyId = null);
```

Comment:
Can we get an explanation of what the `$keyId` is, and why it needs to be passed to the `encode` function?